js-react/README.md

```diff
@@ -162,32 +163,22 @@ function onGuessChange(event) {
 }
 ```
 
-We are now ready to implement the `handleSubmit` function which will process the submitted form data. The function will get, similar to the `onChange` of the input tag, an event object. Normally when you submit a form the form fields will be send to the server, but we want to perform the calculation in the browser so we have to disable the default action with.
-
-```jsx
-event.preventDefault();
-```
-
-Like we did in the previous post we have to construct a web worker.
-
-```jsx
-const worker = new Worker('worker.js');
-```
-
-The `worker.js` is the same as in the previous post so we re-use it by
+We are now ready to implement the `handleSubmit` function which will process the submitted form data. The function will get, similar to the `onChange` of the input tag, an event object. Normally when you submit a form the form fields will be send to the server, but we want to perform the calculation in the browser so we have to disable the default action with `preventDefault()` function. We will then construct a module as we did in the previous post.
 
 ```jsx
-cd react && ln -s ../webassembly/worker.js . && cd -
+function handleSubmit(event) {
+  event.preventDefault();
+  // Wait for module to initialize,
+  createModule().then(({NewtonRaphson}) => {
+    // Perform computation
+    const newtonraphson = new NewtonRaphson(tolerance);
+    const root = newtonraphson.solve(initial_guess);
+    setRoot(root);
+  });
+}
 ```
 
-We have to post a message to the worker with the values from the form.
-
-```jsx
-worker.postMessage({
-  type: 'CALCULATE',
-  payload: { tolerance: tolerance, initial_guess: initial_guess }
-});
-```
+We will use the same `newtonraphson.js` and `newtonraphson.wasm` files as in the [previous post](). Make sure you download these files from [here]().
```

Comment:
This line seems in the wrong place to me. Think it would be better between line 38 and 40 to explain the newtonraphson.js script tag.